index.html

```diff
@@ -75,7 +83,7 @@ <h1 class="works-title">My Recent Works</h1>
             </div>
             
             <div class="first-card">
-                <img class="first-project" src="img/top-proj.png" alt="project-screenshot">
+                <div class="first-project"></div>
 
                 <div class="first-card-content">
```

Comment:
- [ ] Cards are almost there, I would suggest reducing the space between them to make your design more similar to the Figma design:

![Captura de pantalla 2022-03-25 a la(s) 7 51 29 p  m](https://user-images.githubusercontent.com/83717285/160218729-b99670d1-681c-45de-9503-a20dcb6b711d.png)